src/routes/Settings/styles.less

```diff
@@ -438,18 +449,7 @@
             flex-direction: column-reverse;
 
             .side-menu-container {
```

Comment:
remove this too as its already specified with display: none on a higher resolution size

Reply:
451-453